universal-login-commons/test/helpers/SetupSdk.ts

```diff
@@ -1,9 +1,9 @@
-import RelayerUnderTest from 'universal-login-relayer/build/utils/relayerUnderTest';
+const RelayerUnderTest = require('universal-login-relayer/build/utils/relayerUnderTest').default;
```

Comment:
#setupSdk should not be a part of commons, but part of SDK. This is not a "common" functionality. This is strictly SDK functionality. Otherwise, we could just move all code to common :)